eng/pipelines/aggregate-reports.yml

```diff
@@ -17,142 +17,166 @@ pr:
     include:
       - eng/pipelines/aggregate-reports.yml
 
-jobs:
-  - job: GenerateReports
-
-    variables:
-      - template: templates/variables/globals.yml
-
-    pool:
-      vmImage: 'windows-2019'
-
-    timeoutInMinutes: 120
-    steps:
-      - download: net-core
-        artifact: packages
-        patterns: '*'
-
-      - pwsh: |
-          npx cspell lint `
-            --config ./.vscode/cspell.json `
-            --no-must-find-files `
```

Comment:
Looks like you might have changed this or have a bad merge. Take a closer look this diff.